pkgs/top-level/all-packages.nix

```diff
@@ -20509,6 +20509,8 @@ with pkgs;
 
   nix-index = callPackage ../tools/package-management/nix-index { };
 
+  nixPin = callPackage ../tools/package-management/nix-pin { };
```

Comment:
Why not `nix-pin` like the project?

Reply:
Huh, I thought there was a preference for camelCase. After digging a bit I realise the opposite is true, so I've now amended the commit to make it `nix-pin`.